apps/air-discount-scheme/backend/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
     "allowJs": true,
-    "types": ["node", "jest", "./types"]
+    "types": ["node", "jest", "./types"],
+    "strict": true,
+    "strictPropertyInitialization": false
```

Comment:
I had some problems with this, but I managed to make it work. https://mariusschulz.com/blog/strict-property-initialization-in-typescript 

just food for thought

Reply:
Nice, I enabled strictPropertyInitialization again and used ! to ignore certain properties that it should ignore.